ignite/pkg/gacli/gacli.go

```diff
@@ -34,16 +35,15 @@ type Metric struct {
 }
 
 // Send sends metrics to GA.
-func (c *Client) Send(metric Metric) error {
-	v := url.Values{
-		"v":   {"1"},
-		"tid": {c.id},
-		"cid": {metric.User},
-		"t":   {"event"},
-		"ec":  {metric.Category},
-		"ea":  {metric.Action},
-		"ua":  {"Opera/9.80 (Windows NT 6.0) Presto/2.12.388 Version/12.14"},
-	}
+func (c Client) Send(metric Metric) error {
+	v := url.Values{}
+	v.Set("v", "1")
+	v.Set("tid", c.id)
+	v.Set("cid", metric.User)
+	v.Set("t", "event")
+	v.Set("ec", metric.Category)
+	v.Set("ea", metric.Action)
+	v.Set("ua", "Opera/9.80 (Windows NT 6.0) Presto/2.12.388 Version/12.14")
```

Comment:
Let's maybe read the OS info and Ignite info instead?
Like`Ignite CLI v0.XX, arch, OS` and use `runtime.GOOS` and `runtime.GOARCH` (https://pkg.go.dev/runtime#pkg-constants)

Reply:
@michelleellen, do you know if we can collect these metrics?